VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -97,7 +97,10 @@ def get_total_used_size(self):
                 self.logger.log("df return code"+str(df.returncode), True)
                 output = df.stdout.read().decode()
             if sys.version_info > (3,):
-                output = str(output, encoding='utf-8', errors="backslashreplace")
+                try:
```

Comment:
This is not related to this ConfidSeqNo work. but I saw many failures in kusto for this. line 101 throws below error which causes failure of size calculation. and report size as 0.

Unable to fetch total used space with error: decoding str is not supported, stack trace: Traceback (most recent call last):
  File "/var/lib/waagent/Microsoft.Azure.RecoveryServices.VMSnapshotLinux-1.0.9185.0/main/Utils/SizeCalculation.py", line 101, in get_total_used_size
    output = str(output, encoding='utf-8', errors="backslashreplace")
TypeError: decoding str is not supported

Reply:
ok good. 
Will the size calculation work now where it was failing earlier (or we can still get "Output of df command is not in desired format") ?

Reply:
Its not giving any such error on my VM now